packages/application-components/src/components/custom-views/custom-views-selector/custom-views-selector.tsx

```diff
@@ -34,8 +41,68 @@ const Wrapper = styled.div<TWrapperProps>`
 `;
 
 const Container = styled.div`
-  background-color: ${designTokens.colorAccent98};
-  padding: ${designTokens.spacing25} ${designTokens.spacing60};
+  background: linear-gradient(
+      0deg,
+      ${designTokens.colorNeutral95},
+      ${designTokens.colorNeutral95}
+    ),
+    linear-gradient(
+      0deg,
+      ${designTokens.colorNeutral98},
+      ${designTokens.colorNeutral98}
+    );
+  padding: 7px ${designTokens.spacing20};
+  border: 1px solid ${designTokens.colorNeutral95};
+  border-radius: ${designTokens.borderRadius8};
+`;
+
+const hoverAndSelectedStateStyles = css`
+  box-shadow: ${designTokens.shadow1};
+  background-color: ${designTokens.colorSurface};
+  color: ${designTokens.colorPrimary30};
+`;
+
+type TCustomViewSelectorItemProps = {
+  selected?: boolean;
+  label: string;
+  onClick?: (
+    event: MouseEvent<HTMLButtonElement> | KeyboardEvent<HTMLButtonElement>
+  ) => void;
+};
+
+const CustomViewSelectorItem = (props: TCustomViewSelectorItemProps) => {
+  return (
+    <AccessibleButton
+      css={css`
+        display: flex;
+        align-items: center;
+        padding: ${designTokens.spacing20} ${designTokens.spacing25};
+        height: 30px;
+        border-radius: ${designTokens.borderRadius6};
+        color: ${designTokens.colorPrimary};
+        ${props.selected && hoverAndSelectedStateStyles}
+
+        :hover {
+          ${hoverAndSelectedStateStyles};
+        }
+      `}
+      onClick={props.onClick}
+      label={props.label}
+    >
+      <Text.Detail
+        tone="inherit"
+        fontWeight={props.selected ? 'bold' : 'regular'}
+      >
+        {props.label}
+      </Text.Detail>
+    </AccessibleButton>
+  );
+};
+
+const Separator = styled.span`
+  width: 1px;
+  height: 18px;
+  background-color: #cccccc;
```

Comment:
Nit: these values shouldn't be hardcoded

Reply:
Don't we have an existing design token for this color? Why do we need to add a new one?

Reply:
If I'm not mistaken, `#cccccc` translates to `hsl(0, 0%, 80%)`. I don't think we have it available as a token.